### PR TITLE
Tweaked news model

### PR DESCRIPTION
**Pull request by tylersplitt**

### CODE REVIEW

**Comment by mkreiser** [comment]
#35

File: app/models.py [hunk 1]
```diff
@@ -78,3 +78,13 @@ class Result (models.Model):
...
+class News (models.Model):
+  author = models.CharField(max_length = 40)
+  post_subject = models.CharField(max_length = 100)
+  post_text = models.TextField()
+  post_datetime = models.DateTimeField()
+  post_season = models.CharField(max_length = 10, choices = ModelEnums.SEASON_OPTIONS)
+
+
+  def __str__(self):
+    return str(self.author + ': ' + self.post_subject)
```
Comment (mkreiser) [inline app/models.py]: Add date in here as well

File: app/serializers.py [hunk 2]
```diff
@@ -21,3 +21,8 @@ class Meta:
     model = Result
     fields = ('id', 'athlete', 'distanceResult', 'event', 'meet', 'performance', 'result_membership')
     depth = 1
+
+class NewsSerializer(serializers.ModelSerializer):
+  class Meta:
+    model = News
+    fields = ('id', 'author', 'post_subject', 'post_text', 'post_datetime', 'post_season')
```
Comment (mkreiser) [inline app/serializers.py]: Remove post text. Shouldn't filter based on a post's content

**Comment by mkreiser** [comment]
:+1:  after that change

**Comment by tylersplitt** [comment]
I made the change and pushed again

File: app/models.py [hunk 3]
```diff
@@ -78,3 +78,13 @@ class Result (models.Model):
...
+class News (models.Model):
+  author = models.CharField(max_length = 40)
+  post_subject = models.CharField(max_length = 100)
+  post_text = models.TextField()
+  post_datetime = models.DateTimeField()
+  post_season = models.CharField(max_length = 10, choices = ModelEnums.SEASON_OPTIONS)
+
+
+  def __str__(self):
+    return str(self.author + ': ' + self.post_subject + ' - ' + str(self.post_datetime))
```
Comment (mkreiser) [inline app/models.py]: You shouldn't need to wrap this entire thing in a `str()`

File: app/serializers.py (hunk 2 above)
Comment (mkreiser) [inline app/serializers.py]: NP: newline

**Pull request merged**